src/__tests__/config.test.ts

```diff
@@ -1,15 +1,26 @@
 import Config from '../config';
 import { dummyLog } from '../utils/logger';
 
+const fetchSpy = jest.fn();
+
+// @ts-ignore
+jest.mock('node-fetch', () => (...args) => fetchSpy(...args));
+
+beforeEach(() => {
+  fetchSpy.mockClear();
+});
+
 const log = dummyLog();
 
 const importMock = jest.fn();
 jest.mock('import-cwd', () => (path: string) => importMock(path));
 
 describe('loadExtendConfig', () => {
-  test('should throw when no config found', async () => {
+  test('should reject when no config found', async () => {
     const config = new Config(log);
-    expect(() => config.loadExtendConfig('nothing')).toThrow();
+    return expect(config.loadExtendConfig('nothing')).rejects.toBeInstanceOf(
```

Comment:
you might be able to do 

```js
await expect(config.loadExtendConfig('nothing')).rejects.toBeInstanceOf(
```

here instead of the return. async jest testing like this is not too ergonomic lol

Reply:
Ohhh, right. Yeah, it's a bit of a nightmare. We need a lint rule for this 😅